apps/meteor/app/api/server/api.helpers.ts

```diff
@@ -101,3 +102,10 @@ export function checkPermissions(options: { permissionsRequired?: PermissionsReq
 	// If reached here, options.permissionsRequired contained an invalid payload
 	return false;
 }
+
+export function parseDeprecation(methodThis: any, { alternatives, version }: { version: string; alternatives?: string[] }): void {
+	const infoMessage = alternatives?.length
+		? `Please use the alternative(s): ${alternatives?.length === 1 ? alternatives[0] : alternatives?.join(',')}`
```

Comment:
I guess we need an space in the beginning 🤔  

and since at this is point we're sure it is an array, we can just use `.join()` 😉 
```suggestion
		? ` Please use the alternative(s): ${alternatives.join(',')}`
```